API/Services/Tasks/Scanner/Parser/Parser.cs

```diff
@@ -557,7 +561,7 @@ public static class Parser
             MatchOptions, RegexTimeout),
         // Korean Chapter: 제n화 -> Chapter n, 가디언즈 오브 갤럭시 죽음의 보석.E0008.7화#44
         new Regex(
-            @"제?(?<Chapter>\d+\.?\d+)(화|장)",
+            @"제?(?<Chapter>\d+)(회|화|장)",
```

Comment:
Don't remove the support for decimal chapters

Reply:
Sorry. I was confused.
Fixed.